warehouse/integrations/mssql/mssql_test.go

```diff
@@ -52,6 +52,7 @@ func TestIntegrationMSSQL(t *testing.T) {
 		schema                string
 		sourceID              string
 		destinationID         string
+		currentWorkspaceID    string
```

Comment:
```suggestion
		workspaceID    string
```
Could we make it as `workspaceID`? It would be analogous to the other changes.